### DISCUSSION TITLE
user-guide/feed

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#184] [文档评论区]
# user-guide/feed

Feed RSS Feed 是发现新研究的好方法。通过 Feed，您可以订阅来自期刊、网站、出版商、机构、研究小组或其他来源的更新，并快速查找新文章或作品。如果您在出版物的提要中找到想要保存并进一步阅读的项目，只需单击按钮即可将其添加到您的 Zotero 库中。 追踪期刊目录更新 追踪数据库中某些【关键词】相关的文献更新情况 订阅任意网站的内容更新，例...

https://main--zotero-zh.netlify.app/user-guide/feed.html

**Comment by ArwenYu** [comment]
亲爱的大大，请问RSS订阅无法自动和手动更新如何解决呢？

**Comment by cth33122** [comment]
为什么我的zotero7找不到RSS订阅入口？

**Reply by lixiaoqi-LXQ** [reply]
左上角，文件->新建文献库->新建订阅

**Reply by cth33122** [reply]
woc,牛！！

**Comment by daytime001** [comment]
订阅知网期刊RSS到Zotero失败，输入网址后标题和OK选项都是灰色的，无法选中。试了一下订阅其它的可以成功，是知网的原因？
![demo](https://daytime001-picturebed.oss-cn-nanjing.aliyuncs.com/typora/1731382611114.png)

**Reply by dhwtb2009** [reply]
同問，遇到了一樣的問題

**Reply by Huster1993** [reply]
订阅链接中要把navi.cnki.net替换为rss.cnki.net

**Comment by yinghuayv** [comment]
sharestuff 在b站找不到啦

**Comment by yinghuayv** [comment]
请问大佬，zotero7 添加一个将  订阅加入 ‘我的文库’ 时可以选择组的功能

**Comment by Tariya321** [comment]
IEEE旗下的文献订阅似乎存在问题，年初开始出现无法更新的情况。
同样途径尝试nature系列文献是可以正常的订阅更新的。
有木有朋友遇到这个问题？

**Comment by Kueteel** [comment]
求助：有没有遇到过PubMed的feed每天都会断线 旁边有个感叹号
